examples/google-signin/google-signin-iframe.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <script src="https://accounts.google.com/gsi/client"></script>
+  <script>
```

Comment:
We'll want to move most the JS into a .js file we host so allow us to update it

Reply:
Cool! I put a dummy file in there for now. I'll have to update it once it actually is ready.